reports/California.md

```diff
@@ -380,6 +380,44 @@ id: ca-losangeles-26
+
+Police arrested protesters en masse in downtown Los Angeles. Protesters hands were zip tied excessively tightly behind their backs and they were loaded onto buses and driven around the city. Some were taken to a temporary field jail at UCLA's Jackie Robinson stadium; UCLA claims it "failed to stop" LAPD from using the stadium as a "staging area," but was not told about the field jail until after the fact. Arestees were detained for long periods, still in excessively tight zip ties. Social distancing procedures were not observed. Many protesters allege that conditions in the makeshift jails were poor -- detainees denied medical attention and permission to use the restroom, for instance -- and that they were released, many with dead phones, in unfamiliar places in the middle of the night with no way to get home. Two arrestees allege that police prevented them from leaving before the curfew went into effect, then arrested them for violating the curfew. One arrestee alleges that she was twice threatened with guns, that she was sexually assaulted by police, and that police mocked detainees having panic attacks. Police also allegedly arrested at least one homeless person (to whom the curfew was not supposed to apply) for a curfew violation. A lawsuit has been filed in connection with these and other abuses by the National Lawyers' Guild; a separate lawsuit has been filed by one of the arrestees. (See also incidents ca-losangeles-32 (near-identical behavior the following day) and ca-santamonica-2 (similar behavior in Santa Monica).)
+
+tags: abuse-of-power, arrest, homeless, inhumane-treatment, threaten
+
+id: ca-losangeles-19
+
+**Links**
+* [Complaint in NLG lawsuit](https://nlg-la.org/wp-content/uploads/sites/5/2020/06/AMENDED-COMPLAINT-ECF.pdf)
+* [Allegations by protester Laura Montilla](https://www.instagram.com/p/CBBNXXkJs0a/?igshid=jgeposybda4a)
+* [Relevant video](https://www.instagram.com/p/CBEr2emJRyJ/?igshid=oloj7lgtv81t)
+* [Local news coverage mentioning the incident](https://abc7.com/protest-arrest-george-floyd-curfews/6234180/)
+* [LA Times article mentioning both lawsuits](https://www.latimes.com/california/story/2020-06-22/complaints-lapd-abuses-mount-protest-lawsuit)
+* [UCLA disapproves of use of Jackie Robinson stadium for a field jail](https://twitter.com/UCLA/status/1268039753015812096)
+* [First response by UCLA professors, containing further allegations of misconduct by LAPD and LASD](https://twitter.com/ananyaUCLA/status/1267985147225231360)
+* [Second response by UCLA professors](https://drive.google.com/file/d/157MdwMMCEebcUyesO9HMOYK9p21q3gpE/view)
+* https://www.reddit.com/r/LosAngeles/comments/gxi8b2/how_is_noone_talking_about_this_women_from/
+* [Allegation of LAPD cutting someone's hand while trying to remove a zip tie](https://www.reddit.com/r/LosAngeles/comments/gy3e3r/the_lapd_has_been_slicing_into_protesters_hands/ft9so6j/?context=8&depth=9)
```

Comment:
Are you asserting this comment on reddit as evidence? 

I don't think this is sufficient to qualify as evidence.

Reply:
About to edit. Not sure I agree that this isn't evidence -- we occasionally accept descriptions from twitter and instagram as evidence, even when they're accompanied with no picture/video, or one that doesn't directly bear on the truth of their claims. The reddit account making the allegation, u/blackwidowla, doesn't (as far as I can tell) have a history of posting content that looks like trolling or astroturfing. The account is 2 years old, and sometimes mentions being from LA. The (admittedly not very many) posts I looked over didn't mention living anywhere else. If u/blackwidowla is a troll, I admire its dedication to its craft. (Of course, that says nothing about the possibility that u/blackwidowla could be a real Angeleno who is lying. But that's a risk we run with any eyewitness description that doesn't come from someone known to be trustworthy.)

Reply:
That's true, but the insta and Twitter posts are always tied to a real world identity. The other posts here are people with their names out there. And they have also been referenced in news articles, which I assume have a higher standard of due diligence

Reply:
And I don't think it really adds enough to be worth the tricky situation imo

Reply:
Good points. Removed.